content/projects/audi-tt/index.mdx

```diff
@@ -5,66 +5,88 @@ status: "in progress"
 authors:
     - Michael
 ---
-import ProjectHeader from "../../../src/components/project-header"
-import Button from "../../../src/components/button"
-import Timeline from "../../../src/components/timeline"
-import ProjectPosts from "../../../src/components/project-posts"
-import AudiImage from "./audi-tt-2003.gif"
 
-<ProjectHeader image={AudiImage} title="Audi TT Restoration" status="in progress">
-  <Button color="#242424" url="https://github.com/mmcatcd"> GitHub </Button>
-</ ProjectHeader>
-<br />
+import Button from "../../../src/components/button";
+import TimelineItem from "../../../src/components/timeline-item";
+import Timeline from "../../../src/components/timeline";
+import ProjectPosts from "../../../src/components/project-posts";
+import AudiImage from "./audi-tt-2003.gif";
+import Project from "../../../src/components/project";
 
-<div style={{ marginLeft: `175px` }}>
-<Text sx={{ fontWeight: `medium`, fontSize: [3, 4], color: `heading` }}>
-  About
-</Text>
 
----
+<Project
```

Comment:
Some big structural changes here. Basically, I've created a `Project` component that abstracts away all of the indentation shtuff. I think it makes a project `mdx` file look a bit cleaner because that way you don't have to see most of the inner workings of it. Let me know if you think this can be improved or you don't like it.